utils/fasm/test/test_fasm_arch.xml

```diff
@@ -28,20 +28,100 @@
     </tile>
   </tiles>
   <layout>
-    <fixed_layout height="10" width="10" name="test">
+    <fixed_layout height="6" width="6" name="test">
       <perimeter type="io" priority="100">
         <metadata>
           <meta name="type">io</meta>
         </metadata>
       </perimeter>
       <corners type="EMPTY" priority="101"/>
-      <fill type="clb" priority="10">
-        <metadata>
-          <meta name="fasm_prefix">CLB</meta>
-        </metadata>
-      </fill>
-      <col type="EMPTY" startx="6" repeatx="8" starty="1" priority="19"/>
-      <col type="EMPTY" startx="2" repeatx="8" starty="1" priority="19"/>
+
+      <single type="clb" priority="10" x="1" y="1">
+      <metadata> <meta name="fasm_placeholders">
+        LOC=X1Y1
+      </meta> <meta name="fasm_prefix">CLB</meta> </metadata>
```

Comment:
Can we test without a root level block prefix as well?  Because that is what we need.

Reply:
Yes, I've tested it by modifying the test locally and it works correctly. I'll add a test case for that.